Description: Fixes builds agains ruby >= 1.9.2
 Ruby 1.9.2 no longer includes the current directory in the LOAD_PATH
 so we'll just use absolute paths instead.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=676104
Author: John Stamp <jstamp@users.sourceforge.net>
Last-Update: 2012-06-06
--- a/admin/Makefile.rb
+++ b/admin/Makefile.rb
@@ -6,7 +6,7 @@
 
 cwd = File.dirname( __FILE__ )
 require 'find'
-require "#{cwd}/platform.rb"
+require File.expand_path(File.join(File.dirname(__FILE__), 'platform.rb'))
 
 
 ######################################################################### defs
--- a/admin/qpp
+++ b/admin/qpp
@@ -4,7 +4,7 @@
 
 cwd=File.dirname __FILE__
 require 'find'
-require "#{cwd}/findsrc"
+require File.expand_path(File.join(File.dirname(__FILE__), 'findsrc.rb'))
 
 sources = Array.new
 headers = Array.new
